MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -101,32 +105,30 @@ private List<string> GetDecoders(string encoderAppPath)
                 "hevc"
             };
 
-            foreach (var codec in required)
-            {
-                var srch = " " + codec + "  ";
+            var found = required.Where(x => output.IndexOf(x, StringComparison.OrdinalIgnoreCase) != -1);
 
-                if (output.IndexOf(srch, StringComparison.OrdinalIgnoreCase) != -1)
-                {
-                    _logger.LogInformation("Decoder available: " + codec);
-                    found.Add(codec);
-                }
-            }
+            _logger.LogInformation("Available decoders: {Codecs}", found);
 
             return found;
         }
 
-        private List<string> GetEncoders(string encoderAppPath)
+        private IEnumerable<string> GetEncoders(string encoderAppPath)
```

Comment:
Cannot we reduce copy-pasteness here? The only difference between two functions should be ffmpeg option plus the list of codecs to look for.